styles/canvas-lms/catppuccin.user.css

```diff
@@ -56,124 +56,975 @@
+          }
+        }
+      }
+      /* right side */
+      span.css-103zv00-view-flexItem {
+        .css-oydprd-view {
+          border-color: @surface1;
+        }
+        [data-testid="expand-participants-button"] {
+          color: @blue;
+        }
+      }
     }
-    .ic-DashboardCard,
-    .ic-DashboardCard__header_content,
-    .ic-DashboardCard__action-container,
-    .ic-DashboardCard__header-subtitle,
-    .ic-DashboardCard__header-title,
-    .ic-DashboardCard__header-term {
-      background-color: @surface0 !important;
+    /* compose message overlay */
+    span:has(> span[data-testid="compose-modal-desktop"]) {
+      span[data-testid="compose-modal-desktop"] {
+        /* the modal itself */
+        background-color: @surface0;
+        border-color: @surface1;
+        span:has(> label[for^="TextArea"]) {
+          border-color: @surface1;
+          textarea {
+            color: @text;
+          }
+          span[class$="textArea__outline"] {
+            border-color: @accent-color;
+          }
+        }
+        span[data-testid="past-messages"] div[class$="view"] {
+          border-color: @surface1;
+        }
+        button[class$="tag"] {
+          background-color: @surface0;
+          border-color: @surface2;
+          svg {
+            color: @text;
+          }
+        }
+      }
     }
 
-    .ic-app-header__main-navigation,
-    .ic-app-header__logomark-container {
-      background-color: @crust !important;
+    /* COURSE CONTENT (QUIZZES, ETC) */
+    #assignment_show .student-assignment-overview {
+      border-color: @surface1;
     }
 
-    .ic-app-header__menu-list-item.ic-app-header__menu-list-item--active
-      .ic-app-header__menu-list-link {
-      background-color: fadeout(@crust, 0.8);
+    #quiz_show {
+      ul#quiz_student_details {
+        border-color: @surface1;
+      }
+      .alert {
+        background-color: @red;
+        span {
+          color: @base;
+        }
+      }
+      .quiz-submission {
+        background-color: @surface0;
+        .score_value {
+          color: @subtext1;
+        }
+      }
     }
 
-    body {
+    .ic-Table {
+      td,
+      th.ic-Table--header-row {
+        border-color: @surface1;
+      }
+    }
+    table.summary {
+      td, tbody th {
+        background-color: @base;
+      }
+    }
+    ul#question_list {
+      li.marked {
+        /* for some reason canvas's css has left in quotes so this just patches that */
+        background-position: left 3px top 7px;
+      }
+      li i {
+        /* question mark icons */
+        color: @subtext0;
+      }
+    }
+    #questions {
       background-color: @base;
+      .ambrosiaContainerComponent * {
+        color: @text;
+      }
+      .question {
+        border-color: @surface1;
+        &.marked .flag_question {
+          /* the flag image is a png containing multiple flags so can't really svg it */
+          filter: brightness(if(( @color-scheme = dark ), 130%, 100%));
```

Comment:
```suggestion
          filter: brightness(if((@lookup != latte), 130%, 100%));
```

Reply:
We also have better solutions for theming PNGs with filters, provided the image is a single color.